pyanaconda/modules/security/certificates/installation.py

```diff
@@ -68,6 +73,12 @@ def run(self):
 
         Dump the certificates into specified files and directories.
         """
+        if self._phase == INSTALLATION_PHASE_PREINSTALL:
+            if self._payload_type != PAYLOAD_TYPE_DNF:
+                log.debug("Not importing certificates in pre install for %s payload.",
```

Comment:
Maybe "in pre install phase" for consistency ?

Reply:
Updated